contexts/credentials/v2

```diff
@@ -87,6 +87,69 @@
           "@container": "@graph"
         }
       }
+    },
+
+    "DataIntegrityProof": {
+      "@id": "https://w3id.org/security#DataIntegrityProof",
```

Comment:
Link is dead: https://w3id.org/security#DataIntegrityProof

Reply:
is there a corresponding PR on security context?

Reply:
Not yet, no. I was going to do that if/when the Data Integrity JSON-LD Context was merged in the Data Integrity spec repo.

Reply:
There is now: https://github.com/w3c/vc-data-integrity/pull/65